### PR TITLE
CSS & HTML validator fixes

### PR DESCRIPTION
**Pull request by olexandrd**

### CODE REVIEW

**Review by LadaM — DISMISSED** [review]

File: src/css/pagination.css [hunk 1]
```diff
@@ -93,7 +93,7 @@
 
       path {
         fill: var(--dark);
-        fill-opacity: 0.5;
+        opacity: 0.5;
```
Comment (Sysqwerty) [inline src/css/pagination.css]: image

Please check the error from validator

Comment (olexandrd) [reply src/css/pagination.css]: image

**Review by Sysqwerty — DISMISSED** [review]

File: src/css/pagination.css [hunk 2]
```diff
@@ -51,58 +51,58 @@
...
-      border: 1px solid var(--light-grey);
+  .pagination-arrow.disabled {
+    cursor: default;
+    border: 1px solid var(--light-grey);
+  }
 
-      .pagination-link {
-        cursor: default;
-      }
+  .pagination-arrow.disabled .pagination-link {
```
Comment (LadaM) [inline src/css/pagination.css:92]: Is it necessary to write it out like that?

Comment (olexandrd) [reply src/css/pagination.css:92]: Yes
* css validator fails on previous syntax
* need to keep exact classes  declaration (with dots and with/without spaces)

**Review by LadaM — APPROVED** [review]

**Pull request merged**